content/en/error_tracking/frontend/browser.md

```diff
@@ -9,63 +9,310 @@ further_reading:
+
+```javascript
+<script
+    src="https://www.datadoghq-browser-agent.com/us1/v6/datadog-rum.js"
+    type="text/javascript">
+</script>
+<script>
+    window.DD_RUM && window.DD_RUM.init({
+      clientToken: '<CLIENT_TOKEN>',
+      applicationId: '<APP_ID>',
+      // site: '<SITE>',
+      service: '<APP_ID>',
+      env: '<ENV_NAME>',
+      // version: '1.0.0'
+    });
+</script>
+```
+
+The `trackUserInteractions` parameter enables the automatic collection of user clicks in your application. **Sensitive and private data** contained in your pages may be included to identify the elements interacted with.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### TypeScript (optional)
+
+If you are initializing the SDK in a TypeScript project, use the code snippet below. Types are compatible with TypeScript >= 3.8.2.
+
+<div class="alert alert-info"><strong>Note</strong>: For earlier versions of TypeScript, import JavaScript sources and use global variables to avoid any compilation issues.</div>
+
+```javascript
+import '@datadog/browser-rum/bundle/datadog-rum'
+
+window.DD_RUM.init({
+  applicationId: 'XXX',
+  clientToken: 'XXX',
+  site: 'datadoghq.com',
+  trackUserInteractions: true,
+   trackResources: true,
+  ...
+})
+```
+
+### Step 3 - Configure environment and settings
+
+1. In the Environment field, define the environment (`env`) for your application to use [unified service tagging][32].
+2. In the Service field, define the service (`service`) for your application to use [unified service tagging][32].
+3. Set the privacy level for user input. See [Session Replay Browser Privacy Options][16] for more details.
+4. Set a version number (`version`) for your deployed application in the initialization snippet. For more information, see [Tagging][33].
+5. Configure additional parameters as needed. See the [Configuration reference](#configuration-reference) section below for all available options. 
+
+#### Tagging for Error Tracking
+
+These tags (configured in step 3 above) power Error Tracking functionality:
+
+- Filtering and faceting issues by `service` and `env`
+- Cross-product correlation with RUM, Logs, and APM for the same `service`/`env`
+- Matching uploaded source maps through the same `service` and `version` you configure during upload
+
+A service is an independent, deployable code repository that maps to a set of pages:
+
+- If your browser application was constructed as a monolith, your Datadog application has one service name for the application.
+- If your browser application was constructed as separate repositories for multiple pages, edit the default service names throughout the lifecycle of your application.
+
+#### Configuration reference
```

Comment:
I wonder if we should add the configuration reference at the end? it's quite long and steps 4-6 kinda get lost because of it